src/ui/layout/main-navigation.tsx

```diff
@@ -133,9 +131,13 @@ export function MainNavigation(props: NavProps) {
 	const loc = useLocation();
 	const path = () => loc.pathname;
 
+	createEffect(() => {
+		console.log(path());
```

Comment:
Forgotten `console.log`

Reply:
😅  sorry. That seems to be my attention-deficit trademark.